project.v4.0.md

```diff
@@ -0,0 +1,787 @@
+- **Repudiation:** Policy changes may not leave tamper-proof audit trails if governance metadata is not consistently anchored.
+
+### 5. Analytics & Feedback
+- **Differential Privacy Weaknesses:** Poorly tuned ε (epsilon) values can leak sensitive micro-group data.  
+- **Re-identification:** Cross-correlating ideological heatmaps with external datasets could expose dissidents, whistleblowers, or minorities.  
+- **Bias & Manipulation:** Automated summarization could misrepresent minority opinions, shaping perception.
+
+### 6. Network & Privacy Layer
+- **Metadata Leaks:** Even Tor/I2P/mixnets struggle against global passive adversaries. Timing analysis may still correlate activity with individuals.  
+- **Denial of Service:** Federated nodes are choke points; targeted DoS or resource exhaustion could silence whole collectives.  
+- **Elevation of Privilege:** Node operators may exploit admin consoles for surveillance beyond intended scope.
+
+### 7. Supply Chain & Updates
+- **Tampering / Malicious Updates:** If release signing or reproducible builds are not rigorously enforced, adversaries can push backdoored clients.  
+- **Rollback Attacks:** Users may be tricked into older, vulnerable versions without strong anti-rollback enforcement.
+
+---
+
+### Key Risk Archetype Intersections
+- **Dissidents vs Authoritarians:** Surveillance + metadata leaks are existential threats.  
+- **Criminal actors (Mafia, traffickers, terrorists):** They gain little benefit long term, but in the short term can still exploit discussion/vote spaces for recruitment before detection.  
+- **Corporate/State Influence:** Subtle governance takeovers, flooding discourse, and analytics manipulation align with their motivations.  
+- **Technologists/Hackers:** May test and expose vulnerabilities early — constructive if embraced, destructive if ignored.
+
+---
+
+### Takeaway
+Most structural risks are *manageable but non-trivial*: Sybil collusion, metadata leaks, coercion, takeover attempts, and weak DP parameters. The system’s **strength is transparency** — harms are visible even if not always preventable. Its **weakness is metadata** — timing, correlation, and coercion can undermine anonymity despite cryptographic protections.
+
+## K) References
+
+[1] Yu, H., Gibbons, P., Kaminsky, M., Flaxman, A. **SybilGuard: Defending Against Sybil Attacks via Social Networks.** SIGCOMM ’06. (PDF). — **Mature**.  
+[2] Yu, H., et al. **SybilLimit: A Near‑Optimal Social Network Defense against Sybil Attacks.** IEEE S&P ’08. (PDF). — **Mature**.  
+[3] Danezis, G., Mittal, P. **SybilInfer: Detecting Sybil Nodes using Social Networks.** NDSS ’09. (PDF). — **Mature**.  
+[4] Cao, Q., et al. **SybilRank: Aiding the Detection of Fake Accounts in Large Scale OSNs.** NSDI ’12. — **Mature**.  
+[5] Viswanath, B., et al. **An Analysis of Social Network‑based Sybil Defenses.** SIGCOMM CCR ’10. — **Mature**.  
```

Comment:
validated ref + link (citation remains)
Analysis of 1-3, but not 4. Also has a "SumUp" defense included - we don't have a reference for SumUp but if the analysis shows it superior then we'll need to include it directly.